# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dovecot/dovecot-2.2-0001-a7d0bb66639f.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---


# HG changeset patch
# User Timo Sirainen <tss@iki.fi>
# Date 1365961247 -10800
# Node ID a7d0bb66639f956329916020ca6717b8840f842d
# Parent  4af6b464ef9badeb0aca695f84b4f695ad4f4b15
Linking error fix.

diff -r 4af6b464ef9b -r a7d0bb66639f src/doveadm/doveadm-mail.h
--- a/src/doveadm/doveadm-mail.h	Fri Apr 12 21:19:38 2013 +0300
+++ b/src/doveadm/doveadm-mail.h	Sun Apr 14 20:40:47 2013 +0300
@@ -150,6 +150,6 @@
 extern struct doveadm_mail_cmd cmd_mailbox_subscribe;
 extern struct doveadm_mail_cmd cmd_mailbox_unsubscribe;
 extern struct doveadm_mail_cmd cmd_mailbox_status;
-struct doveadm_mail_cmd cmd_batch;
+extern struct doveadm_mail_cmd cmd_batch;
 
 #endif

